src/popup/components/AddressFormatted.vue

```diff
@@ -21,6 +21,7 @@ export default defineComponent({
   props: {
     address: { type: String, required: true },
     columnCount: { type: Number, default: 6 },
+    splitAddress: Boolean,
```

Comment:
maybe "protocol" prop is more suitable

Reply:
It's not about protocol - I made it as temporary solution until we have proper address validation for all protocols